CUSW-chula/cusw-frontend
# Fix/banyaphon real time collaboration

## Description

- **Related Issues:**  Connects to #62
- **Type of Change:** New feature / Refactoring

## Changes Made

- [x] Split the BlockNote and Workspace components.
- [x] Integrated Y-Sweet Provider for real-time collaboration.
- [x] Setup delay for patch description 

## How to Test

*Provide step-by-step instructions on how to verify the changes. Include any setup details or special conditions that need to be met.*

1. Ensure the Y-Sweet Provider is connected properly and that real-time collaboration works.

## Screenshots (if applicable)

<img width="1265" alt="image" src="https://github.com/user-attachments/assets/bd9ac7ac-83de-491a-bbeb-988ca36e4637" />

## Checklist


## Additional Notes

With BlockNote and Workspace separated, real-time collaboration (using Y-Sweet Provider) becomes easier to implement and manage.

---
